util/ksonnet/ksonnet.go

```diff
@@ -119,7 +119,16 @@ func (k *ksonnetApp) Show(environment string) ([]*unstructured.Unstructured, err
 	if err != nil {
 		return nil, fmt.Errorf("`ks show` failed: %v", err)
 	}
-	return kube.SplitYAML(out)
+	data, err := kube.SplitYAML(out)
+	if err == nil {
+		for _, d := range data {
+			err = kube.UnsetLabel(d, "ksonnet.io/component")
```

Comment:
Can we reference the bug in a comment so future readers understand the reasoning?